deployment/README.md

```diff
@@ -40,13 +39,13 @@ bastion_ami           = "ami-08f3d892de259504d"
 bastion_instance_type = "t3.nano"
 bastion_ebs_optimized = true
 
-rds_database_identifier = floodmap-staging
+rds_database_identifier = floodmap
 rds_database_name       = floodmap
 rds_database_username   = floodmap
 rds_database_password   = floodmap
 ```
 
-This file lives at `s3://noaafloodmap-staging-config-us-east-1/terraform/terraform.tfvars`.
+This file lives at `s3://noaafloodmap-config-us-east-1/terraform/terraform.tfvars`.
```

Comment:
The old state and variables files have been moved over and the old staging bucket has been removed.